test/parallel/test-child-process-exec-error.js

```diff
@@ -3,6 +3,11 @@ var common = require('../common');
 var assert = require('assert');
 var child_process = require('child_process');
 
+if (common.isWindows) {
+  common.skip('symlinks are weird on windows');
+  return;
+}
```

Comment:
(Looks like this was meant for `test-process-execpath.js` and not `test-child-process-exec-error.js`?)